### PR TITLE
Add more shopping_list tests

### PR DESCRIPTION
**Pull request by karawoo**
Filling in some more of our pending tests for `/shopping_list` to return 200 and give an empty shopping list when we haven't created any list items.

### CODE REVIEW

File: spec/shopping_list_spec.rb [hunk 1]
```diff
@@ -6,8 +6,15 @@
...
-  it 'returns 200'
-  it 'gives an empty shopping list'
+  it 'returns 200' do
+    get '/shopping_list'
+    expect(last_response.status).to eq 200
+  end
+
+  it 'gives an empty shopping list' do
+    get '/shopping_list'
+    expect(ListItem.all).to be_empty
```
Comment (no-reply) [inline spec/shopping_list_spec.rb:16]: we agreed that, even though this reaches into the model layer, this is the best test for this behavior right now.

we could consider rewriting once some details about the HTML rendering are done
